extensions/measurement-tracking/src/panels/PanelStudyBrowserTracking/PanelStudyBrowserTracking.tsx

```diff
@@ -135,6 +135,7 @@ export default function PanelStudyBrowserTracking({
       customMapDisplaySets={mapDisplaySetsWithTracking}
       onClickUntrack={onClickUntrack}
       onDoubleClickThumbnailHandlerCallBack={checkDirtyMeasurements}
+      onClickThumbnailHandlerCallBack={checkDirtyMeasurements}
```

Comment:
I don't believe there is a need for two callbacks here. Perhaps just refactor to one callback and call it something appropriate, say `onThumbnailActivatedHandler1 or somethinig.